samples/cpp/stitching_detailed.cpp

```diff
@@ -813,7 +817,7 @@ int main(int argc, char* argv[])
             }
             blender->prepare(corners, sizes);
         }
-        else if (!timelapser)
+        else if (!timelapser && timelapse)
```

Comment:
If you mean that there is too many parameters in this sample I 'm 
agree.  I propose to close this pull request and make two new pull 
request one without timelapser and another with timelapser parameter 
only (with fixed parameter only). If you are agree I can manage first 
example and may be  @*pglotov <https://github.com/pglotov> * can manage 
other PR

Le 25/10/2015 11:51, Alexander Alekhin a écrit :
>
> In samples/cpp/stitching_detailed.cpp 
> <https://github.com/Itseez/opencv/pull/4110#discussion_r42943606>:
>
> > @@ -813,7 +817,7 @@ int main(int argc, char* argv[])
> >              }
> >              blender->prepare(corners, sizes);
> >          }
> > -        else if (!timelapser)
> > +        else if (!timelapser && timelapse)
>
> This sample is rather old and big (~1k lines). A log of changes have 
> been added here (includes possible caching for Blender object, which 
> looks curiously now).
> I believe it will be good to have separate PRs with new changes (this 
> PR already has many fixes)
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Itseez/opencv/pull/4110/files#r42943606>.
>